tests/unit/components/cylc/toolbar.vue.spec.js

```diff
@@ -31,7 +30,30 @@ Vue.use(Vuex)
 describe('Toolbar component', () => {
   let vuetify
   let $route
-  const store = new Vuex.Store(storeOptions)
+  // for some obscene reason, using the actual "options.js" store, gets contaminated data from other tests during the course of the whole test running
```

Comment:
Is this something that you think would need fixing/further investigation; I'm still a bit novice with the ui - I would guess yes? If so, I could open an issue.